src/test/java/com/puppycrawl/tools/checkstyle/internal/powermock/MainFrameModelPowerTest.java

```diff
@@ -84,28 +77,4 @@ public void testShouldAcceptFile() throws IOException {
                 MainFrameModel.shouldAcceptFile(nonExistentFile));
     }
 
-    @Test
-    public void testOpenFileWithUnknownParseMode() throws CheckstyleException {
```

Comment:
> remove MainFrameModelPowerTest.testOpenFileWithUnknownParseMode because it tests a language construct

I am not sure what you mean. Most likely this test is needed for code coverage. CI will let you know.

Reply:
The test checks that something happens if the `enum` would be extended. I assume: Since its state can't change after compile time the coverage checker knows that the branch can't be reached and therefore can be excluded from coverage. I'm sure that after the PMD exclusions are added, CI passes and that this is correct.

Reply:
> Since its state can't change after compile time the coverage checker knows that the branch can't be reached and therefore can be excluded from coverage

Code coverage is never that smart. This is what prompted us to add stuff like this.
Code coverage is passing so I can only assume the code has changed since this test was added.

Reply:
My point is that the test doesn't test any logic. It tests that Java enums work. You want to make sure that you don't forget to handle the case after adding a value to the enum, but that's not what unit tests are for. And you'd spot it through other tools altering after a line coverage decrease.

Ususally I agree with "code coverage is never that smart", but I think here this isn't the case.

After all, the test doesn't do any harm, however I have no idea how to adapt it to OpenJDK 12 which is the goal here.

Reply:
such tests exists only for test coverage, is code is changed and such test is not required, lets remove it.